src/wallet/test/rpc_wallet_tests.cpp

```diff
@@ -1128,7 +1128,7 @@ BOOST_AUTO_TEST_CASE(rpc_z_sendmany_parameters)
 
     BOOST_CHECK_THROW(CallRPC("z_sendmany"), runtime_error);
     BOOST_CHECK_THROW(CallRPC("z_sendmany toofewargs"), runtime_error);
-    BOOST_CHECK_THROW(CallRPC("z_sendmany just too many args here"), runtime_error);
+    BOOST_CHECK_THROW(CallRPC("z_sendmany addr [] 1 0.001 true"), runtime_error);
```

Comment:
This is no longer checking that too many args results in a runtime error. I think it's instead checking that the `addr` field is invalid?